deps/v8/src/mirror-debugger.js

```diff
@@ -1515,7 +1515,7 @@ PropertyMirror.prototype.name = function() {
 
 
 PropertyMirror.prototype.toText = function() {
-  if (IS_SYMBOL(this.name_)) return %SymbolDescriptiveString(this.name_);
+  if (IS_SYMBOL(this.name_)) return %SymbolDescription(this.name_);
```

Comment:
Yes. %_ValueOf is only needed for object wrappers around primitives. Being a property key, it cannot be a wrapper here.